about.html

```diff
@@ -0,0 +1,139 @@
+    crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="./style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300&display=swap"
+    rel="stylesheet">
+  <title>About Page</title>
+</head>
+
+<body>
+  <div class="top-panel">
+    <ul class="top-text d-flex ms-2 text-white me-5 pe-5 list-unstyled justify-content-center mt-1">
+      <li class="me-3"><i class="fa-brands fa-facebook-f"></i></li>
+      <li class="me-3"><i class="fa-brands fa-twitter"></i></li>
+      <li class="me-3"><a class="text-decoration-none text-white" href="">English</a></li>
+      <li class="me-3"><a class="text-decoration-none text-white" href="">My Page</a></li>
+      <li class="me-3"><a class="text-decoration-none text-white" href="">Log Out</a></li>
+    </ul>
+  </div>
+
+  <nav class="nav" id="nav">
+    <div class="container">
+      <div class="nav-container-parnav">
+        <div class="nav-container-parnav-brand display">
+          <a href="index.html">
+            <img src="./images/mylogo.png" alt="sasas" class="responsive" />
+          </a>
+        </div>
+        <ul class="nav-ul display">
+          <li><a href="index.html" class="colored">Home</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponser</a></li>
+          <li><a href="#">News</a></li>
+          <li class="parbordered">
+            <a href="#" class="bordered">Uni Camping</a>
+          </li>
+        </ul>
+        <div class="ham">
+          <div class="bar bar-one"></div>
+          <div class="bar bar-two"></div>
+          <div class="bar bar-three"></div>
+        </div>
+      </div>
+    </div>
+  </nav>
+
+  <section class="top-section about d-flex flex-column justify-content-center mt-5">
+    <h1 class="first-text about-first px-2 h4">"Hello! Web Developers."
+    </h1>
+    <h2 class="about-topic align-items-center px-2">WEB DEVELOPMENT GLOBAL SUMMIT 2023
+    </h2>
+
+    <p class="card p-3 topic-description-about mx-3 mt-5">
+      Our Web Development Company brings the creative minds together with the right skills, set to work together and
+      transform their
+      ideas into the reality that can contribute towards the development of the IT industry.
+    </p>
+
+    <p class="topic-description contact-us text-center px-5 mt-5 pt-5">
+      Please contact us via following Email for any
+      further questions about Web Development company
+    </p>
+    <p class="text-center mail"><a href="mailto:Web-Development@outlook.com">Web-Development@outlook.com</a></p>
+  </section>
+
+  <section class="card p-lg-3 pb-lg-5 border-1 m-0">
+    <h2 class="logo-text text-center px-4 py-4 h4">Web Development Company Bootcamp <br>2023 Logo</h2>
```

Comment:
- [ ] After the title we have a divider, please kindly add it 🙃 🙃 

Your design

<img width="436" alt="Screenshot 2023-03-23 at 18 55 06" src="https://user-images.githubusercontent.com/74501523/227266122-32c2edae-2af3-4b25-add1-22d092343f06.png">

Original design

<img width="436" alt="Screenshot 2023-03-23 at 18 55 53" src="https://user-images.githubusercontent.com/74501523/227266437-8fb01ace-b65f-4b48-b859-40de219449b5.png">